# Car steering and throttle commands.

# Value of the steering command [-1,1]
# 1.0 = max left, -1.0 = max right
float32 steering

# Value of the throttle command [-1,1]
# 1.0 = max throttle, -1.0 = max brake
float32 throttle
